tests/contracts/provider_contracts_test.py

```
@@ -70,6 +70,52 @@ def test_native_health_allows_omitted_server_id(self):
+        requirements = {
+            requirement["id"]: requirement
+            for requirement in self.valid_data["nativeSiloContract"]["requirements"]
+        }
+        expected_routes = {
+            "native.playback.capability": ("GET", "/api/v1/playback/capability"),
+            "native.playback.start-legacy": ("POST", "/api/v1/playback/start"),
+            "native.playback.progress": ("POST", "/api/v1/playback/{session_id}/progress"),
+            "native.playback.stop": ("DELETE", "/api/v1/playback/{session_id}"),
+            "native.playback.track": ("PATCH", "/api/v1/playback/{session_id}/audio"),
+        }
+        for contract_id, route in expected_routes.items():
```

Ruff (0.16.1)

[warning] Loop control variable `route` not used within loop body

Rename unused `route` to `_route`

(B007)